### DISCUSSION TITLE
[Feature] Choose the backup quality of your photos & videos

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#4601] [Feature Request]
### The feature

You can say that the title speaks for itself and most of those who are here at immich probably arrived via google photos
The proposal is to implement the option that appears on the [original Google page regarding different storage methods](https://support.google.com/photos/answer/6220791)

I wonder if this possibility exists or will exist because
a family member of mine likes to take a lot of pictures (and most of the time they will be the same picture) Is there an option in immich to "compress items" like there is in Google Photos?
This option could be great, especially if it would be possible to run it per user and not on all photos (just like in Google Photos)

I also confirm that I checked [Roadmap](https://github.com/orgs/immich-app/projects/1) and found no ticket for this

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by aviv926** [comment]
I see someone started something about it here #1242

also someone made something that already work with immich

[https://gist.github.com/JamesCullum/6604e504318dd326a507108f59ca7dcd](https://gist.github.com/JamesCullum/6604e504318dd326a507108f59ca7dcd)

**Comment by bo0tzz** [comment]
Duplicate of #1202
